common/script/content/appearance/chair.js

```diff
@@ -0,0 +1,5 @@
+import prefill from './prefill.js';
+
+export default prefill({
+   black: {}
```

Comment:
Yes, "black wheelchair" is a good default!

On Thu, Mar 3, 2016 at 7:34 PM, Sinble <notifications@github.com> wrote:

> In common/script/content/appearance/chair.js
> <https://github.com/HabitRPG/habitrpg/pull/6816#discussion_r54986890>:
>
> > @@ -0,0 +1,5 @@
> > +import prefill from './prefill.js';
> > +
> > +export default prefill({
> > +   black: {}
>
> I did color because of @lemoness <https://github.com/lemoness> statement
> of "(since we will probably have other colors eventually)."
> Let me know
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/6816/files#r54986890>.
>